infrastructure/proto/log_replication_metadata.proto

```diff
@@ -25,3 +25,23 @@ message LogReplicationMetadataKey {
 message LogReplicationMetadataVal {
   string val = 1;
 }
+
+/*
+ * Replication Status Key
+ */
+ message ReplicationStatusKey {
+   enum Status {
+     REPLICATION_COMPLETION = 0;
+     IS_DATA_CONSISTENT = 1;
+   }
+   Status key = 1;
+ }
+
+/*
+ * Replication Status Value
+ * Active Site sets the completionPercent, Standby sets the dataConsistent boolean
+ */
+message ReplicationStatusVal {
+  uint64 replicationCompletion = 1;
+  bool dataConsistent = 2;
```

Comment:
should we also add an enum which indicates if we are in LOG_ENTRY (delta) or SNAPSHOT (full) sync? for completeness.